gradle/libs.versions.toml

```diff
@@ -1,11 +1,11 @@
 [versions]
 agp = "8.2.0"
-kotlin = "2.0.21"
+kotlin = "2.1.0"
 multiplatform-settings = "1.2.0"
-android-minSdk = "21"
+android-minSdk = "26"
```

Comment:
@RafaRuiz is looking into the implications of this, but in the meantime you maybe able to get away with desugaring https://developer.android.com/studio/write/java8-support#library-desugaring